ocpp/charge_point.py

```diff
@@ -140,12 +140,11 @@ def __init__(self, id, connection, response_timeout=30):
             return
 
         if msg.message_type_id == MessageType.Call:
-            # Call's can be validated right away because the 'action' is know.
+            # Call's can be validated right away because the 'action' is known.
```

Comment:
This comment seems a little out of place to me since we moved the validation step.

Reply:
Good catch!